src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -111,7 +111,28 @@ private static MIBreakpointState StringToBreakpointState(string addr)
 
             string basename = System.IO.Path.GetFileName(documentName);     // get basename from Windows path
             basename = process.EscapePath(basename);
-            BindResult bindResults = EvalBindResult(await process.MICommandFactory.BreakInsert(basename, line, condition, ResultClass.None), pbreak);
+
+            Checksum[] checksums = null;
+#if CORECLR
```

Comment:
Why is `#if CORECLR` needed here, given that it's checking `MICommandFactory.SupportsBreakpointChecksums()`.